webpack.config.js

```diff
@@ -30,6 +30,7 @@ const webpack = require("webpack");
 module.exports = {
     entry: './src/index.ts',
     devtool: 'source-map',
+    mode: 'development',
```

Comment:
remove it

Reply:
Was used for debugging tests. Now removed.